enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -622,58 +622,178 @@ will be reconciled in any namespaces, removing the need for ServiceMeshMemberRol
 Cluster-scoped mode is a Tech Preview feature in OSSM 2.3 and fully supported in
 OSSM 2.4.
 
-#### Security Policy
+#### Gateway Topology
+
+Users have the option to deploy their Gateways using two distinct topologies: shared gateways or dedicated gateways.
+Each topology addresses specific customer requirements, comes with different security implications (see [Security Policy](#security-policy)),
+and will be supported in OpenShift.
+
+##### Shared Gateway Topology
+
+With shared gateway topology, a single load balancer type service and proxy deployment is created and serves
+routes across multiple namespaces. The Gateway must allow the desired application namespaces via the
+`spec.listeners.allowedRoutes.namespaces` field. This topology mirrors the existing OpenShift router topology and
+is ideal for users that are cost sensitive and do not have strict performance or security requirements.
+
+```mermaid
+flowchart TD
+    subgraph foo-app namespace
+        xRoute
+    end
+    subgraph bar-app namespace
+        xRoute2
+    end
+    subgraph openshift-ingress namespace
+        Gateway
+        ProxyDeployment
+        LBService
+    end
+    Gateway[Gateway] -.-> ProxyDeployment(Proxy Deployment)
+    Gateway[Gateway] -.-> LBService(LB Service)
+    xRoute[xRoute] --> Gateway[Gateway]
+    xRoute2[xRoute] --> Gateway[Gateway]
+```
 
-TBD.
+This cross-namespace topology can only be implemented using the Gateway-to-xRoute hierarchy, as Istio does not support
+cross-namespace Gateway merging. Therefore, a limitation of this topology is that xRoutes lack an API for
+specifying TLS certificates, requiring all application certificates to be managed within a cluster-wide Gateway.
+
+Users can have multiple shared Gateways, similar to the concept of [sharding](https://docs.openshift.com/container-platform/latest/networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-ingress-controller.html#nw-ingress-sharding_configuring-ingress-cluster-traffic-ingress-controller) in OpenShift. 
+
+##### Dedicated Gateway Topology
+
+With a dedicated gateway topology, a load balancer type service and proxy deployment is created to serve the xRoutes
+in a dedicated namespace. This approach is more suitable for multi-tenant environments or applications that require a
+dedicated gateway to meet specific security or performance requirements.
```

Comment:
As far as I know, we don't really have any data that says dedicated gateways are more secure or better performing, so it's best not to mention that here or in our docs.

Reply:
I think the security comes from restricting who can use the certificates on a gateway and from having a dedicated proxy and load-balancer for each gateway.